src/pages/publication/PublicationList.tsx

```diff
@@ -42,57 +43,78 @@ interface PublicationListProps {
-              </StyledTableCellForDate>
-              <TableCell>
-                <Button
-                  color="primary"
-                  component={RouterLink}
-                  to={`/publication/${publication.id}`}
-                  data-testid={`edit-publication-${publication.id}`}>
-                  <StyledEditIcon />
-                  <NormalText>{t('common:edit')}</NormalText>
-                </Button>
-              </TableCell>
-              <TableCell>
-                <Button color="secondary" variant="outlined" data-testid={`delete-publication-${publication.id}`}>
-                  <StyledDeleteIcon />
-                  {t('common:remove')}
-                </Button>
-              </TableCell>
-            </StyledTableRow>
-          ))}
-      </TableBody>
-    </StyledTable>
+    <>
+      <StyledTable>
+        <TableHead>
+          <TableRow>
+            <TableCell>
+              <Label>{t('workLists:publication_name')}</Label>
+            </TableCell>
+            <TableCell>
+              <Label>{t('common:status')}</Label>
+            </TableCell>
+            <TableCell>
+              <Label>{t('common:date')}</Label>
+            </TableCell>
+            <TableCell />
+            <TableCell />
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {publications
+            .filter(publication => publication.status !== PublicationStatus.PUBLISHED)
+            .map(publication => (
+              <StyledTableRow key={publication.id}>
+                <TableCell component="th" scope="row">
+                  <NormalText>{publication.title}</NormalText>
+                </TableCell>
+                <StyledTableCellForStatus>
+                  <NormalText>{t(`publication:status.${publication.status}`)}</NormalText>
+                </StyledTableCellForStatus>
+                <StyledTableCellForDate>
+                  <NormalText>{publication.createdDate}</NormalText>
+                </StyledTableCellForDate>
+                <TableCell>
+                  <Button
+                    color="primary"
+                    component={RouterLink}
+                    to={`/publication/${publication.id}`}
+                    data-testid={`edit-publication-${publication.id}`}>
+                    <StyledEditIcon />
+                    <NormalText>{t('common:edit')}</NormalText>
+                  </Button>
+                </TableCell>
+                <TableCell>
+                  <Button
+                    color="secondary"
+                    variant="outlined"
+                    data-testid={`delete-publication-${publication.id}`}
+                    onClick={() => handleOnClick(publication)}>
+                    <StyledDeleteIcon />
+                    {t('common:remove')}
+                  </Button>
+                </TableCell>
+              </StyledTableRow>
+            ))}
+        </TableBody>
+      </StyledTable>
+      {deletePublicationId && (
```

Comment:
har allerede en sjekk for om modal skal vises i `DeletePublicationModal.tsx` (`openModal`)